versions/kruise/1.8.0/templates/rbac_role.yaml

```diff
@@ -988,6 +992,9 @@ rules:
   - clonesets/rollback
   - statefulsets
   - statefulsets/scale
+  - statefulsets/rollback
```

Comment:
i don't think cloneset and statefulsets has subresource for rollback.

Reply:
GET